src/drivers/nuopc/cime_flds/esmFlds.F90

```diff
@@ -1970,577 +1441,563 @@ subroutine esmFlds_Init(gcomp, rc)
-    !    call fld_add(flds_x2o, flds_x2o_map,'Foxx_rofi_HDO', longname=longname, stdname=stdname, units=units)
-    !    call fld_add(flds_r2o_ice, flds_r2o_ice_map,'Forr_rofi_HDO', longname=longname, stdname=stdname, units=units)
+    !    call fld_add(flds_r2x, flds_r2x_map,'Forr_rofi_HDO')
+    !    call fld_add(flds_x2o, flds_x2o_map,'Foxx_rofi_HDO')
+    !    call fld_add(flds_r2o_ice, flds_r2o_ice_map,'Forr_rofi_HDO')
 
     !    ! r2x, x2l
 
     !    units    = 'kg m-2 s-1'
     !    longname = 'H2_16O waterrflux due to flooding'
     !    stdname  = 'H2_16O_flodding_water_flux_back_to_land'
-    !    call fld_add(flds_r2x, flds_r2x_map,'Flrr_flood_16O', longname=longname, stdname=stdname, units=units)
+    !    call fld_add(flds_r2x, flds_r2x_map,'Flrr_flood_16O')
     !    call fld_add(flds_x2l, flds_x2l_map,'Flrr_flood_16O')
 
     !    longname = 'H2_18O waterrflux due to flooding'
     !    stdname  = 'H2_18O_flodding_water_flux_back_to_land'
-    !    call fld_add(flds_r2x, flds_r2x_map,'Flrr_flood_18O', longname=longname, stdname=stdname, units=units)
+    !    call fld_add(flds_r2x, flds_r2x_map,'Flrr_flood_18O')
     !    call fld_add(flds_x2l, flds_x2l_map,'Flrr_flood_18O')
 
     !    longname = 'HDO Waterrflux due to flooding'
     !    stdname  = 'HDO_flodding_water_flux_back_to_land'
-    !    call fld_add(flds_r2x, flds_r2x_map,'Flrr_flood_HDO', longname=longname, stdname=stdname, units=units)
+    !    call fld_add(flds_r2x, flds_r2x_map,'Flrr_flood_HDO')
     !    call fld_add(flds_x2l, flds_x2l_map,'Flrr_flood_HDO')
 
     !    longname = 'H2_16O river channel water volume '
     !    stdname  = 'H2_16O_rtm_volr'
-    !    call fld_add(flds_r2x, flds_r2x_map,'Flrr_volr_16O', longname=longname, stdname=stdname, units=units)
+    !    call fld_add(flds_r2x, flds_r2x_map,'Flrr_volr_16O')
     !    call fld_add(flds_x2l, flds_x2l_map,'Flrr_volr_16O')
 
     !    longname = 'H2_18O river channel water volume '
     !    stdname  = 'H2_18O_rtm_volr'
-    !    call fld_add(flds_r2x, flds_r2x_map,'Flrr_volr_18O', longname=longname, stdname=stdname, units=units)
+    !    call fld_add(flds_r2x, flds_r2x_map,'Flrr_volr_18O')
     !    call fld_add(flds_x2l, flds_x2l_map,'Flrr_volr_18O')
 
     !    longname = 'HDO river channel water volume '
     !    stdname  = 'HDO_rtm_volr'
-    !    call fld_add(flds_r2x, flds_r2x_map,'Flrr_volr_HDO', longname=longname, stdname=stdname, units=units)
+    !    call fld_add(flds_r2x, flds_r2x_map,'Flrr_volr_HDO')
     !    call fld_add(flds_x2l, flds_x2l_map,'Flrr_volr_HDO')
 
     !    ! longname = 'H2_18O Waterrflux due to flooding'
     !    ! stdname  = 'H2_18O_flodding_water_flux_back_to_land'
-    !    ! call fld_add(flds_r2x, flds_r2x_map,'Flrr_flood_HDO', longname=longname, stdname=stdname, units=units)
+    !    ! call fld_add(flds_r2x, flds_r2x_map,'Flrr_flood_HDO')
     !    ! call fld_add(flds_x2l, flds_x2l_map,'Flrr_flood_HDO')
 
```

Comment:
I'm not sure why you changed all of this stuff in comments - can it just be removed?